pkgs/by-name/holo-cli/default.nix

```diff
@@ -0,0 +1,19 @@
+{
+  pkgs ? import <nixpkgs> {
+    overlays = [
+      (import (fetchTarball "https://github.com/oxalica/rust-overlay/archive/master.tar.gz"))
+    ];
```

Comment:
If we wanted to, we could either apply the overlay directly to [our pkgs](https://github.com/ngi-nix/ngipkgs/blob/bbf2cdaa6ba3d6c6f576d188df34cc8c508292ab/default.nix#L8) or even create a new set of pkgs with the overlay applied, but we don't really need to use this at all.

We also want to avoid importing random inputs and instead add them to our [flake inputs](https://github.com/ngi-nix/ngipkgs/blob/bbf2cdaa6ba3d6c6f576d188df34cc8c508292ab/flake.nix#L4-L17), both to keep track of what we use and to properly pin its version.

PS: Always prefer [pinning the versions](https://nix.dev/reference/pinning-nixpkgs.html) whenever you're fetching something. Fetching the latest one is bad for reproducibility as things might break upstream.